lib/index.ts

```diff
@@ -22,53 +25,44 @@ export interface DepTree {
   _isProjSubpkg?: boolean;
 }
 
-export function inspect(root, targetFile) {
+interface CountDict {
+  [k: string]: number;
+}
+
+export async function inspect(root, targetFile) {
 
-  return Promise.all([
+  const result = await Promise.all([
```

Comment:
Can you please remove the `Promise.all` while you are in here?

Reply:
It's there for a reason: to perform these tasks in parallel. A small saving, but still.